scripts/modules/dataTweaks.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
```

Comment:
Thought I'd move these into their own file as they were dotted throughout the cleanup script.